modulefiles/module_base.noaacloud.lua

```diff
@@ -0,0 +1,51 @@
+help([[
+Load environment to run GFS on noaacloud
+]])
+
+local spack_mod_path=(os.getenv("spack_mod_path") or "None")
+prepend_path("MODULEPATH", spack_mod_path)
+
+load(pathJoin("stack-intel", (os.getenv("stack_intel_ver") or "None")))
+load(pathJoin("stack-intel-oneapi-mpi", (os.getenv("stack_impi_ver") or "None")))
+load(pathJoin("python", (os.getenv("python_ver") or "None")))
+
+--load(pathJoin("hpss", (os.getenv("hpss_ver") or "None")))
+load(pathJoin("gempak", (os.getenv("gempak_ver") or "None")))
+load(pathJoin("ncl", (os.getenv("ncl_ver") or "None")))
```

Comment:
@weihuang-jedi The `NCL` error you are seeing can be remedied by deleting this line:
```suggestion
```

Reply:
Got NCL-version removed.